rfcs/065-library-data-link-explorer/README.md

```diff
@@ -0,0 +1,128 @@
+# RFC 065: Library Data Link Explorer
+
+
+# Library Data Link Explorer
+
+## Overview
+
+This RFC outlines the plan for the Library Data Link Explorer web application. This tool will enable Collections Information colleagues the ability to view and debug work relationships independently, potentially replacing the workflow of requesting a developer-run script to produce a matcher graph .dot file. 
+
+![Example DOT Notation Graph](https://hackmd.io/_uploads/ry15cEQmkl.png)
+*Example .dot graph returned by Matcher logic.*
```

Comment:
Detail but it renders like this? 
https://github.com/wellcomecollection/docs/blob/065-rfc-ldle/rfcs/065-library-data-link-explorer/README.md
<img width="638" alt="Screenshot 2024-11-26 at 13 31 03" src="https://github.com/user-attachments/assets/d0278d74-2912-4f33-85c4-3b2ed48afc3f">